components/ruby/config/product_info.toml

```diff
@@ -1,4 +1,3 @@
-# DO NOT CHANGE - overwritten by expeditor at build time
```

Comment:
This is no longer a true statement?

Reply:
In fact, it is still a true statement - this is a misleading changeset. There is an expeditor [configuration](https://github.com/chef/license-acceptance/blob/06c48a3baff7ba37de29164fb8bb88e49deb53c0/.expeditor/config.yml#L41) that copies /product_info.toml to components/ruby/config/product_info.toml. The [script](https://github.com/chef/license-acceptance/blob/master/.expeditor/copy_product_info.sh) also inserts the warning line. What has happened here is that Tyler has manually copied the file to sync it, and it thus does not have the warning line.